mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -2359,6 +2370,11 @@ static void getAttentionTypes(SmallVectorImpl<Type> &result,
         MemRefType::get(currentSeqDims, elemTypes[currentSeqLenIndex]);
     result.push_back(currSeqLenType);
   }
+  if (returnLSE) {
+    SmallVector<int64_t> lseDims{groupSize * numHeadsQ, sequenceLengthQ};
+    MemRefType lseType = MemRefType::get(lseDims, elemTypes[lseIndex]);
+    result.emplace_back(lseType);
```

Comment:
Rest of the places are using `result.push_back()`. For consistency change this to use `push_back` too or change all to use `emplace_back`